.github/workflows/ci.yml

```diff
@@ -44,8 +47,21 @@ jobs:
       - run: npm ci
       - run: npm run build
       - run: npm run coverage
+      - name: Check conditions for upload
+        id: check-conditions
+        run: |
+          # We can only upload coverage report if origin repo for PR is same as target repo
+          if [[ ("${{ github.event_name }}" == "pull_request") && \
+                ("${{ github.event.pull_request.head.repo.name }}" == "${{ github.repository}}") ]]; then
+            echo "publish_comment=true" >> $GITHUB_OUTPUT
+          else
+            echo "publish_comment=false" >> $GITHUB_OUTPUT
+          fi
+        shell: bash
 
-      - uses: 5monkeys/cobertura-action@v13
+      - name: Add comment to Pull Request
+        if: ${{ steps.check-conditions.publish_comment == 'true' }}
+        uses: 5monkeys/cobertura-action@v14
```

Comment:
in some other repositories we already switched to adding the report into the workflow summary,
see https://github.com/eclipse-velocitas/devenv-devcontainer-setup/blob/main/.github/workflows/ci.yml#L96 for example
the 2 last steps are needed to replace that one

we also don't need the check than.

Reply:
Sounds like a good plan